src/plugins/LADTable/components/LadTableSet.vue

```diff
@@ -0,0 +1,146 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+<template>
+    <table class="c-table c-lad-table">
+        <thead>
+            <tr>
+                <th>Name</th>
+                <th>Timestamp</th>
+                <th>Value</th>
+            </tr>
+        </thead>
+        <tbody>
+            <template
+                v-for="primary in primaryCollection">
+                <tr class="c-table__group-header"
+                    :key="primary.key">
+                    <td colspan="10">{{primary.domainObject.name}}</td>
+                </tr>
+                <lad-row
+                    v-for="secondary in secondaryCollection[primary.key]"
+                    :key="secondary.key"
+                    :domainObject="secondary.domainObject">
+                </lad-row>
+            </template>
+        </tbody>
+    </table>
+</template>
+
+<style lang="scss">
+
+</style>
+
+<script>
+    import LadRow from './LadRow.vue';
+
+    export default {
+    inject: ['openmct', 'domainObject'],
+    components: {
+        LadRow
+    },
+    data() {
+        return {
+            primaryCollection: [],
```

Comment:
can we name these more clearly?  it's not clear what they represent or how they relate to a "LAD table"